Framework/Algorithms/src/NormaliseToMonitor.cpp

```diff
@@ -384,10 +384,10 @@ void NormaliseToMonitor::checkProperties(
   Property *monID = getProperty("MonitorID");
   // Is the monitor spectrum within the main input workspace
   const bool inWS = !monSpec->isDefault();
-  m_syncScanInput = inputWorkspace->detectorInfo().isSyncScan();
+  m_scanInput = inputWorkspace->detectorInfo().isScanning();
```

Comment:
Minor point: Seems a bit strange to set members variables in an algorithm's `exec`. Could just be declared local and passed around.

Reply:
@OwenArnold This is already a big PR, I would avoid cleaning up unrelated code.